<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarControllBoard</class>
 <widget class="QWidget" name="CarControllBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>223</width>
    <height>621</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="carsActionGbx">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>223</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="straightBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7d7d7d;;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:5px;
border:solid 2px black;</string>
    </property>
    <property name="text">
     <string>直行</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#7d7d7d;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7d7d7d;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:5px;
border:solid 2px black;</string>
    </property>
    <property name="text">
     <string>左转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7d7d7d;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>右转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7d7d7d;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:5px;
border:solid 2px black;</string>
    </property>
    <property name="text">
     <string>退格</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7d7d7d;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actionControlBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: #7d7d7d;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="lightsStatusGbx">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>223</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carLightSetBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:#d77b31;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beeSwitchBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #d77b31;
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>蜂鸣器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="leftCbx">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>车前左大灯</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rightCbx">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>车前右大灯</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="roofRedCbx">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>车顶红色灯</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="roofBlueCbx">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>车顶蓝色灯</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="locationMsgGbx">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>223</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="xAxisLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X轴坐标:0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="yAxisLbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y轴坐标:0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="roadNumberLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>路号:0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="directionLbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>方向:l-&gt;r</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3号小车</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2号小车</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4号小车</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1号小车</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="actionLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>223</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #7d7d7d;
color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-top-right-radius:15px;</string>
   </property>
   <property name="text">
    <string>车辆行为控制</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="componentLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#d77b31;
color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-top-right-radius:15px;</string>
   </property>
   <property name="text">
    <string>车辆组件控制</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="locationLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>419</y>
     <width>223</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #7d7d7d;
color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-top-right-radius:15px;</string>
   </property>
   <property name="text">
    <string>车辆位置信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="locationLbl_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>223</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #7d7d7d;
color: rgb(255, 255, 255);
border-bottom-left-radius:15px;
border-bottom-right-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
